default/scripting/species/SP_EXOBOT.focs.txt

```diff
@@ -15,16 +15,15 @@ Species
     ]
 
     effectsgroups = [
-        [[AVERAGE_INDUSTRY]]
+        [[BAD_INDUSTRY]]
         [[BAD_RESEARCH]]
         [[NO_STOCKPILE]]
 
         [[AVERAGE_POPULATION]]
         [[AVERAGE_HAPPINESS]]
-        [[AVERAGE_SUPPLY]]
+        [[BAD_SUPPLY]]
         [[BAD_DEFENSE_TROOPS]]
         [[BAD_OFFENSE_TROOPS]]
-        [[BAD_WEAPONS]]
```

Comment:
You must drop the corresponding tag as well.

Reply:
Is this even intentional? Undocumented change in initial post.

Reply:
They for sure should be bad pilots. Must be an error.

Reply:
fixed.